src/SWP/WebRendererBundle/Entity/PageRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Web Renderer Bundle
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\WebRendererBundle\Entity;
+
+/**
+ * PageRepository
+ */
+class PageRepository extends \Doctrine\ORM\EntityRepository
```

Comment:
I would create generic EntityRepository where PageRepository will extend it. Then we will have more control over the code, and it will be easier to add some generic functions to the repositories, eg. pagination, some custom, generic functions (e.g. diffrent than findOneBy etc) to fetch the data etc.